lib/std/rand.zig

```diff
@@ -776,6 +773,22 @@ pub const Gimli = struct {
     }
 };
 
+/// Cryptographically secure random number generator based on system calls.
+/// This is just a wrapper for `std.crypto.randomBytes()` that implements the `Random` interface.
+pub const OsRandom = struct {
+    random: Random,
+
+    pub fn init() OsRandom {
+        return OsRandom{
+            .random = Random{ .fillFn = fill },
+        };
+    }
+
+    fn fill(r: *Random, buf: []u8) void {
+        std.crypto.randomBytes(buf) catch @panic("System random number generator not available");
```

Comment:
Hmm, this makes me think `std.crypto.randomBytes` is badly location; perhaps should have its implementation moved here.

Reply:
Having it in `std.crypto` also makes sense. Maybe we can reexport it.